src/cli.js

```diff
@@ -39,6 +39,8 @@ export default function CLI(argv, cwd) {
     .help("h").alias("h", "help")
     .version().alias("v", "version")
     .wrap(cli.terminalWidth())
+    .strict()
+    .showHelpOnFail(false)
```

Comment:
Thanks for the feedback!
It even returns `A command is required.` when given an incorrect argument like `lerna nope`. But it seems `yargs` has no way of preventing that.